src/coq_elpi_builtins.ml

```diff
@@ -3382,6 +3382,21 @@ Supported attributes:
        !: t)),
   DocAbove);
 
+   MLCode(Pred("coq.reduction.lazy.whd_betaiota_deltazeta_for_iota_state",
+    CIn(term,"T",
+    COut(term,"Tred",
+    Read(proof_context, {|Puts T in weak head normal form.
```

Comment:
I have no clue what it does, but it does not do "just whd" ;-)

Reply:
Indeed, thank you. Is there a difference between "weak head normal form" and "head normal form"? If so, the documentation of `whd_betaiota_deltazeta_for_iota_state` in `reductionops.ml` states "head normal form", while the function name starts with "whd", is that a typo?

Reply:
Also, would you like for me to PR this commit (after fixing the doc) to coq-elpi now?